src/model/user.model.ts

```diff
@@ -76,10 +76,10 @@ export class User {
     @prop({ required: true, default: 0 })
     credit: number;
 
-    @prop({ default: "" })
+    @prop()
     evidenceInfo?: String;
```

Comment:
`String` should be `string` since it refers to a type string not a class String.

Reply:
Copy that